aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -252,11 +252,19 @@ bool check_requires_grad_and_head_dim_gt192_and_sm_ge86_lt90(
   auto dprops = at::cuda::getCurrentDeviceProperties();
   bool is_sm86_or_sm89 = check_sm_version<sm86, sm89>(dprops);
   bool is_head_dim_gt192 = params.query.sym_size(-1) > 192;
-  if (input_requires_grad(params) && is_sm86_or_sm89 && is_head_dim_gt192) {
+  bool is_head_dim_lte224 = params.query.sym_size(-1) <= 224;
```

Comment:
so now it's `gt192_lte224`?

Reply:
yeah talked with Tri about this, the condition is interesting:
but essentially any input in (192,224] will error because this requires 7 warps which is not divisible by 4

For things greater than 224 we round up to 256 and this is 8 warps, but if you use dropout you use too much shmem so can't run